### PR TITLE
Add some details in README.md

### PR DESCRIPTION
**Pull request by htjjen**
- Add fields listed in the job.
- List the languages searched.

Issue #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
+Let's find good first issues to contribute OSS. 
+
+This is a simple script to list issues labelled as "good first issue" in repositories with over 500 stars. This script lists the repository name, owner, and URL for each issue. The languages searched include the following:
+* JavaScript
+* TypeScript
+* Ruby
+* Go
+* Java
+* Python
+* Shell
```
Comment (ohbarye) [inline README.md]: @htjjen 
Actually, this job doesn't limit language to the 7 above. A language can vary as long as GitHub can recognize the value.

You can find which language we can specify on GitHub trending page. Then you see there're quite many languages.

image

So my suggestion is, 

- Stop listing up languages
- Explain which language is available

What do you think?

Comment (htjjen) [reply README.md]: @ohbarye That makes sense! 
Perhaps I can change it to "Any language that can be recognized by GitHub can be searched." ?

Comment (ohbarye) [reply README.md]: That sounds 🆒

Comment (htjjen) [reply README.md]: I've committed the change and sent the PR. Thank you!

**Comment by htjjen** [comment]
@ohbarye Thank you!

**Pull request merged**